bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -3454,18 +3410,14 @@ private void drawAsync_530_BarGraph(final GC gcGraph, final GraphDrawingData dra
             /*
              * make sure the bars do not overlap
              */
-            if (serieLayout != ChartDataYSerie.BAR_LAYOUT_SINGLE_SERIE) {
-               if (devXPosNextBar > 0) {
-                  if (devXPos < devXPosNextBar) {
+            if (serieLayout != ChartDataYSerie.BAR_LAYOUT_SINGLE_SERIE && devXPosNextBar > 0 && devXPos < devXPosNextBar) {
 
```

Comment:
Why not keep a better readable code

![mt unformatted code](https://user-images.githubusercontent.com/1283445/109531443-2ba90380-7ab8-11eb-9c7d-e480f92c716c.png)

Reply:
Reverted